mitmproxy/options.py

```diff
@@ -70,6 +70,7 @@ def __init__(
             mode = "regular",  # type: str
             no_upstream_cert = False,  # type: bool
             rawtcp = False,  # type: bool
+            websockets=False,  # type: bool
```

Comment:
All the other lines in this block use spaces around the `=`. Would you mind fixing it please to keep consistency?